include/fast_float/ascii_number.h

```diff
@@ -47,6 +64,40 @@ uint64_t read_u64(const char *chars) {
   return val;
 }
 
+#ifdef FASTFLOAT_SSE2
+
+fastfloat_really_inline
+uint64_t simd_read8_to_u64(const __m128i data) {
+FASTFLOAT_SIMD_DISABLE_WARNINGS
+  static const char16_t kmasks[] = { 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff };
+  const __m128i masks = _mm_loadu_si128(reinterpret_cast<const __m128i*>(kmasks));
+
+  // todo: with AVX512BW + AVX512VL, can use cvtepi16_epi8 instead of masking + pack
+  __m128i masked = _mm_and_si128(data, masks);
```

Comment:
Why are you masking the data here? If the numbers are all digits, then they are all ASCII, correct? In which case, what is the masking doing?

Reply:
Oops, that's a mistake. You're right, it's not necessary.